### PR TITLE
NOTICKET: Updating README with ways to add dictionary values, track time of flight

### PR DESCRIPTION
**Pull request by KunjPrasad**
Updating README with ways to add dictionary values and track time of flight

### CODE REVIEW

**Comment by KunjPrasad** [comment]
@russelljk : Sorry, not sure how to add you as reviewer. First time.. let me know please if I can do anything else on my end to help

File: README.md [hunk 1]
```diff
@@ -101,6 +101,40 @@ In `url.py`, register 'ptrack.urls' on your desired url prefix pattern:
...
+However, doing above requires explicity writing out all key value pairs. If the template context contains a field having dictionary type data (example: `dict_field = {"df_key1": "df_value", "df_key2": 2}`), or field that references a model object with a property method that returns a dictionary type data (example: `model_field = model`, such that `model.dict_property` returns `{"model_key1": "model_key", "model_key2": 3}`), then it is possible to collect these values within `ptrack` by passing them as positional arguments to the `ptrack` template tag, like:
+```
+{% ptrack dict_field model_field.dict_property key1=value1 key2=value2 %}
+```
+Additionally, the `CustomTrackingPixel.record` method must be modified to read the positional arguments and include them with the keyworded arguments, like:
+```
+import ptrack
+class CustomTrackingPixel(ptrack.TrackingPixel):
+    def record(self, request, *args, **kwargs):
+        log.info(request.META['HTTP_USER_AGENT'])
```
Comment (russelljk) [inline README.md:118]: nitpick: You can safely leave this line out of the readme I think.

**Review by russelljk — APPROVED** [review]
Looks good, I left a small nitpick but I'm ok to merge this if you are ready.

File: README.md (hunk 1 above)
Comment (KunjPrasad) [reply README.md]: Thanks. Made the change. Removed this line, and one more line later under the for loop over dictionary value entries

**Review by russelljk — APPROVED** [review]
lgtm

**Pull request merged**